jung-visualization/src/main/java/edu/uci/ics/jung/visualization/PluggableRenderContext.java

```diff
@@ -375,6 +376,15 @@ public void setVertexLabelTransformer(Function<? super N, String> vertexLabelTra
     this.vertexLabelTransformer = vertexLabelTransformer;
   }
 
+  public void setVertexLabelDrawPaintTransformer(
+      Function<? super N, Paint> vertexLabelDrawPaintTransformer) {
+    this.vertexLabelDrawPaintTransformer = vertexLabelDrawPaintTransformer;
+  }
+
+  public Function<? super N, Paint> getVertexLabelDrawPaintTransformer() {
+    return vertexLabelDrawPaintTransformer;
+  }
+
```

Comment:
The old way to set the color for the labels was vv.setForeground(Color)..... Not exactly intuitive! I added a transformer to address this specifically and also allowed it to be a requested property,